propertyestimator/workflow/groups.py

```diff
@@ -362,14 +390,17 @@ def _try_merge_protocol(self, other_protocol_id, other_group, merged_ids, parent
             raise RuntimeError('A protocol with id {} has already been merged '
                                'into the group.'.format(other_protocol_id))
 
-        protocols = self._root_protocols if parent_id is None else self._dependants_graph[parent_id]
+        protocol_ids = self._root_protocols if len(parent_ids) == 0 else []
```

Comment:
Will `parent_ids` be a single `str` (per docstring) or `List[str]`? This could be tough to debug if someone misinterprets, because both are index-able.

Reply:
Typo - great catch!